dev-db/etcd/files/etcd-3.5.9-respect-go-env-vars.patch

```diff
@@ -0,0 +1,33 @@
+--- a/build.sh
++++ b/build.sh
+@@ -18,7 +18,7 @@ GOARCH=${GOARCH:-$(go env GOARCH)}
+ # Set GO_LDFLAGS="-s" for building without symbols for debugging.
+ # shellcheck disable=SC2206
+ GO_LDFLAGS=(${GO_LDFLAGS:-} "-X=${VERSION_SYMBOL}=${GIT_SHA}")
+-GO_BUILD_ENV=("CGO_ENABLED=0" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS:-}" "GOOS=${GOOS}" "GOARCH=${GOARCH}")
++GO_BUILD_ENV=("CGO_ENABLED=0" "GO_BUILD_FLAGS=${GO_BUILD_FLAGS:-}" "GOOS=${GOOS}" "GOARCH=${GOARCH}" "GO111MODULE=${GO111MODULE}" "GOCACHE=${GOCACHE}" "GOMODCACHE=${GOMODCACHE}")
```

Comment:
I would expect all of these environment variables to pass through if they are exported, since I don't see `env --ignore-environment` used. Can you explain why you needed this patch?

Reply:
I added etcd-3.5.10 without this patch. GOCACHE seems to be working for me (`${T}/go-build` contains lots of stuff), so let me know if you have an issue.

Reply:
yeah they patched 3.5.10, finally!
We don't need it anymore. Thanks